hollow/src/main/java/com/netflix/hollow/tools/stringifier/HollowRecordJsonStringifier.java

```diff
@@ -49,6 +51,18 @@
     private final boolean collapseAllSingleFieldObjects;
     private final boolean prettyPrint;
 
+    private Map<Character, String> escapedSymbols = Collections.unmodifiableMap(
```

Comment:
Two parallel arrays might be more efficient. The map is definitely more readable, but again we might be in an area where some slightly less readable optimizations are appropriate.